linked-list.js

```diff
@@ -0,0 +1,169 @@
+import assert from "assert";
+
+export function createList(array = []) {

```

Comment:
naming this `array` is confusing, can call it as `arrayOrList` instead.  Also de

Reply:
Implemented the changes.